src/core/p5.Renderer2D.js

```diff
@@ -1385,13 +1385,18 @@ p5.Renderer2D.prototype._applyTextProperties = function() {
 
 p5.Renderer2D.prototype.push = function() {
   this.drawingContext.save();
+
+  // get the base renderer style
+  return p5.Renderer.prototype.push.apply(this);
 };
 
-p5.Renderer2D.prototype.pop = function() {
+p5.Renderer2D.prototype.pop = function(style) {
```

Comment:
I understand what's happening here, but a comment about what style is would be helpful for future devs.